### PR TITLE
[SV] added variations

### PR DESCRIPTION
**Pull request by edgein666**

### CODE REVIEW

File: sentences/sv/_common.yaml [hunk 1]
```diff
@@ -59,13 +59,15 @@ expansion_rules:
...
   ljusintensitet: "(ljusstyrka | intensitet | styrka | nivå | effekt)"
   ljuskällor: "(ljuskällor[na] | ljus[en | et] | [glöd|led|diod]lamp[ a | an | or | orna] | lysrör[en| et] | belysning[en])"
-  dimra: "(sätt | vrid | dra [på] | skruva [på] | dimra | dimma | ställ [in] | höj | sänk)"
+  dimra: "(ändra | sätt | vrid | dra [på] | skruva [på] | dimra | dimma | ställ [in] | höj | sänk)"
   gardiner: "([rull | motor]gardin[er | erna | en] | persienn[er | erna | en] | spjälgardin[er | erna | en] | draperi[er | erna | en]) | förhänge[n | na]"
   stäng_gardiner: "(stäng [igen | till]| dra (ner | igen | för | till ) | hissa ner | veckla (ner | igen) | tillslut) | slå (igen |ihop | för)"
   öppna_gardiner: "(öppna [upp] | dra (upp | bort | isär | undan) | hissa upp | veckla upp)"
   varmt_kallt: "(varmt | varm | kallt | kall | het | svalt )"
   temperature: "{temperature} [grader] [{temperature_unit}]"
+  change: "(färga | ändra | byt)"
```
Comment (Robin-St) [inline sentences/sv/_common.yaml:68]: I think it’s better to call this byta_färg or similar as they are specific to change of colour and not generic changes.

Comment (larsdunemark) [reply sentences/sv/_common.yaml:68]: Agree, `färga` might also work as alias.

File: sentences/sv/light_HassLightSet.yaml [hunk 2]
```diff
@@ -13,9 +13,9 @@ intents:
           name: "all"
 
       - sentences:
-          - "(färga | ändra | byt) <name> [till] {color}"
+          - "(<change>) <name> [till] {color}"
```
Comment (larsdunemark) [inline sentences/sv/light_HassLightSet.yaml:16]: I think that we can add:
`kan du byta färg på sänglampan till rött`
Since `kan du` is added as skip_word it enables an additional sentence.

File: sentences/sv/_common.yaml (hunk 1 above)
Comment (edgein666) [reply sentences/sv/_common.yaml:68]: I was under the impression that we where to use English variables names?

Comment (larsdunemark) [reply sentences/sv/_common.yaml:68]: @edgein666 I got an review comment about this in the beginning telling me that we should keep `expansion_rules` in Swedish unless the just wraps fields like `name` or 'area`
An right now most is in Swedish so we can try keep it like that and if we want to change it in the future it easy to replace all `expansion_rules`.

Comment (Robin-St) [reply sentences/sv/_common.yaml:68]: Färga sounds good.

File: sentences/sv/light_HassLightSet.yaml (hunk 2 above)
Comment (Robin-St) [reply sentences/sv/light_HassLightSet.yaml:16]: As a test or in the sentence?

Comment (larsdunemark) [reply sentences/sv/light_HassLightSet.yaml:16]: I think that as test and adding sentence to match it. Will be some more options to the sentence.

Comment (larsdunemark) [reply sentences/sv/light_HassLightSet.yaml:16]: `(<change>)` should not have `()` around since it's already in the expansion_rules.

Comment (edgein666) [reply sentences/sv/light_HassLightSet.yaml]: > I think that we can add: `kan du byta färg på sänglampan till rött` Since `kan du` is added as skip_word it enables an additional sentence.

Can you add this separately?

> `(<change>)` should not have `()` around since it's already in the expansion_rules.

removed

Comment (larsdunemark) [reply sentences/sv/light_HassLightSet.yaml]: sure, lets add it as a new MR.

File: sentences/sv/light_HassLightSet.yaml [hunk 3]
```diff
@@ -13,9 +13,9 @@ intents:
           name: "all"
 
       - sentences:
-          - "(färga | ändra | byt) <name> [till] {color}"
+          - "<färga> <name> [till] {color}"
 
       - sentences:
-          - "{color} ljus i <area>"
+          - "[<change>] [till ] {color} ljus i <area>"
```
Comment (larsdunemark) [inline sentences/sv/light_HassLightSet.yaml:19]: `<change>` doesn't exist anymore, shouldn't it be changed to `färga`?

Comment (edgein666) [reply sentences/sv/light_HassLightSet.yaml:19]: Ah, dang, missed it, will fix tomorrow!

Comment (edgein666) [reply sentences/sv/light_HassLightSet.yaml]: I think we should be good to go now.

**Comment by Robin-St** [comment]
@larsdunemark It seems like the automatic tests are not running for this PR, do you see anything to approve this to run? The first run of test of a new contributor seems to be started manually. 

@edgein666 Have you got the environment set up to run the tests locally? Usually good to add some test sentences for new words to try catch as much as possible :) But the code and logic seems good to me.

**Comment by edgein666** [comment]
@Robin-St yep, figured out how to do it now. 🙂

**Comment by edgein666** [comment]
[SV] Added "dim to half"
This was not meant to be added now. Will see if I can remove it.

**Comment by edgein666** [comment]
Not able to remove my latest commit, "dim to half", I wanted to discuss it first but managed to commit anyway.

**Comment by edgein666** [comment]
thanks @larsdunemark, I tried, are we good to go now?

File: sentences/sv/light_HassLightSet.yaml [hunk 4]
```diff
@@ -46,9 +46,41 @@ intents:
...
+          brightness: 100
+
+      - sentences:
+          - "<minimal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+          - "[<dimra>] [ner] [<alla>] <ljuskällor> till <minimal> [<möjliga>] <i_på> <area>"
+          - "[<dimra>] [ner] [<alla>] <ljuskällor> <i_på> <area> till <minimal> [<möjliga>]"
+          - "[ner] till <minimal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+        slots:
+          name: "all"
+          brightness: 1
```
Comment (larsdunemark) [inline sentences/sv/light_HassLightSet.yaml:79]: This looks like duplication from line 29.
It might be a merge problem?

**Review by larsdunemark — CHANGES_REQUESTED** [review]
It's a little hard to follow the MR when adding different changes in the same MR.
In this case it seems like there is a merge conflict adding the same sentences twice.

A good practice is to add a description in the MR about what is suppose to be added so it is easier to see if something is added by mistake.

File: sentences/sv/light_HassLightSet.yaml (hunk 4 above)
Comment (Robin-St) [reply sentences/sv/light_HassLightSet.yaml:79]: Looks merge problems, it seems like these yaml files with a lot of "- sentences:" can really confuse git.

Comment (edgein666) [reply sentences/sv/light_HassLightSet.yaml:79]: sorry for the delay, been away this week. should I just close this PR and create new ones with only one change in each?

Comment (Robin-St) [reply sentences/sv/light_HassLightSet.yaml:79]: I think that is easiest. Pull the latest changes to main to your fork and then create new branches from main where you apply one change :)

**Comment by edgein666** [comment]
will create new PR.
